linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/rule.yml

```diff
@@ -53,20 +53,22 @@ references:
     stigid@ol8: OL08-00-020353
     stigid@rhel8: RHEL-08-020353
 
-ocil_clause: 'the above command returns no output, or the umask is configured incorrectly'
+ocil_clause: 'the value for the "umask" parameter is not "{{{ xccdf_value("var_accounts_user_umask") }}}", or the "umask" parameter is missing or is commented out'
 
 ocil: |-
-    Verify the <tt>umask</tt> setting is configured correctly in the <tt>{{{ etc_bash_rc }}}</tt> file by
-    running the following command:
-    <pre>$ sudo grep "umask" {{{ etc_bash_rc }}}</pre>
-    All output must show the value of <tt>umask</tt> set as shown below:
-    <pre>umask {{{ xccdf_value("var_accounts_user_umask") }}}</pre>
+    Verify the <tt>umask</tt> setting is configured correctly in the <tt>{{{ etc_bash_rc }}}</tt> file with the following command:
+
+    Note: If the value of the "umask" parameter is set to "000" "/etc/bashrc" file, the Severity is raised to a CAT I.
```

Comment:
With this specific STIG language should we move this content to the `checktext` key?